### DISCUSSION TITLE
PHP 8.2 Mailphrase

### DISCUSSION THREAD

**Discussion by RautShlok** [uvdesk/community-skeleton#720] [Q&A]
Hi,
 
is there a mailphrase for php8.2?

My UVDesk is running on the latest version of xampp with php 8.2.

**Comment by PeopleInside** [comment]
Hi, @RautShlok , I hope your day is going well.

> is there a mailphrase for php8.2?

Yes should be for 8.2 also for PHP 8.3 but on XAMPP (Windows) you can have issues as the mailphrase can not support PHP 8.2 on Windows.

On server side (Linux) I had no issue in the past running mailphrase on PHP 8.2 or 8.3

I made a video guide in the past
https://forums.uvdesk.com/topic/2106/how-to-generate-uvdesk-zip-or-install-on-localhost-with-xampp

that with time was no more working due to issue with PHP and mailphrase on XAMPP for Windows but you can try to see if the issue was resolved so support was implemented.

https://forums.uvdesk.com/topic/2157/getting-an-error-when-i-try-to-install-on-localhost
